src/__experimental__/components/tooltip/tooltip.component.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledTooltipInner, StyledTooltipWrapper, StyledTooltipPointer } from './tooltip.style';
+import tagComponent from '../../../utils/helpers/tags';
+
+class Tooltip extends React.Component {
+  static propTypes = {
+    /**
+     * Sets alignment of pointer on tooltip
+     * Options: top, bottom, center, right, left
+     */
+    align: PropTypes.string,
```

Comment:
you should use `alignAroundEdges` from the `OptionsHelper` and remove these options from the comment.